radio/src/gui/128x64/view_channels.cpp

```diff
@@ -39,23 +38,20 @@ void menuChannelsView(event_t event)
     case EVT_KEY_FIRST(KEY_LEFT):
       secondPage = !secondPage;
       break;
-#ifdef MIXERS_MONITOR
     case EVT_KEY_FIRST(KEY_ENTER):
       mixersView = !mixersView;
       break;
-#endif
   }
 
   if (secondPage)
     ch = 16;
   else
     ch = 0;
 
-#ifdef MIXERS_MONITOR
   if (mixersView)
   lcdDrawTextAlignedCenter(0*FH, MIXERS_MONITOR);
   else
-#endif
+
   lcdDrawTextAlignedCenter(0*FH, CHANNELS_MONITOR);
```

Comment:
Sorry, I didn't realize `lcdDrawTextAlignedCenter` was a macro.  If you put the conditional in parenthesis, it works (tested on AVR compiler).  I guess that's a good reason to always use brackets in such cases (note to self).  

`lcdDrawTextAlignedCenter(0, (mixersView ? MIXERS_MONITOR : CHANNELS_MONITOR));`

(Not saying you need to change this, but just FYI).